src/FormElement/ScheduleElement/WeeklyFields.php

```diff
@@ -95,6 +95,29 @@ public function loadWeekDays(array $days): array
         return $values;
     }
 
+    public function validate()
```

Comment:
Isn't it possible to achieve this by a default validator? I don't quite like that you're overriding `validate()`.